backend/pkg/config/config.go

```diff
@@ -37,6 +37,8 @@ const (
 	StateBackupFileName          = "STATE_BACKUP_FILE_NAME"
 	NoMonitorEnvVar              = "NO_K8S_MONITOR"
 	K8sLocalEnvVar               = "K8S_LOCAL"
+	DiffsSendIntervalMin         = "DIFFS_SEND_INTERVAL"
```

Comment:
better to call it DiffsSendIntervalSec and also DIFFS_SEND_INTERVAL_SEC 
and adjust:
ticker := time.NewTicker(time.Duration(interval) * time.Minute) to seconds